.github/workflows/deployDoc.yml

```diff
@@ -22,12 +22,18 @@ jobs:
     - name: Configure CMake
       run: cmake -B "${{github.workspace}}/build" -DHUB_BUILD_DOC=ON
       
-    - name: Build doxygen doc
-      run: cmake --build ${{github.workspace}}/build --target hub-doc-doxygen
+    - name: Install Sphinx and Breathe
+      run:
+        sudo apt install -y sphinx-doc
+        && pip3 install sphinx-rtd-theme breathe sphinx-sitemap
+
+    - name: Build sphinx doc
+      run:
+        cmake --build ${{github.workspace}}/build --target hub-doc-sphinx
 
     - name: Deploy doc 🚀
       uses: JamesIves/github-pages-deploy-action@v4
       with:
-        folder: ${{github.workspace}}/build/doc/docs_doxygen/html # The folder the action should deploy.
+        folder: ${{github.workspace}}/doc/sphinx/_build/html/ # The folder the action should deploy.
```

Comment:
```suggestion
        folder: ${{github.workspace}}/build/doc/sphinx/_build/html/ # The folder the action should deploy.
```

Reply:
The compilation of sphinx imply doxygen compilation, because sphinx uses doxygen doc.

Reply:
The sphinx doc is compiled locally. I am going to make compilation in build folder.

Reply:
Done, check the file.